geode-for-redis/src/test/java/org/apache/geode/redis/internal/data/RedisStringTest.java

```diff
@@ -113,6 +113,82 @@ public void appendStoresStableDelta() {
     assertThat(stringOne.hasDelta()).isFalse();
   }
 
+  @Test
+  public void setStoresStableDelta() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
+    final byte[] baseBytes = {'0', '1'};
+    final byte[] bytesToSet = {'2', '3'};
+
+    when(region.put(any(), any()))
+        .thenAnswer(invocation -> validateDeltaSerialization(baseBytes, invocation));
+    RedisString stringOne = new RedisString(baseBytes);
+
+    stringOne.set(region, null, bytesToSet, null);
+
+    verify(region).put(any(), any());
+    assertThat(stringOne.hasDelta()).isFalse();
+  }
+
+  @Test
+  public void incrStoresStableDelta() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
+    final byte[] baseBytes = {'1'};
+
+    when(region.put(any(), any()))
+        .thenAnswer(invocation -> validateDeltaSerialization(baseBytes, invocation));
+    RedisString stringOne = new RedisString(baseBytes);
+
+    stringOne.incr(region, null);
+
+    verify(region).put(any(), any());
+    assertThat(stringOne.hasDelta()).isFalse();
+  }
+
+  @Test
+  public void incrbyStoresStableDelta() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
+    final byte[] baseBytes = {'1'};
+
+    when(region.put(any(), any()))
+        .thenAnswer(invocation -> validateDeltaSerialization(baseBytes, invocation));
+    RedisString stringOne = new RedisString(baseBytes);
+
+    stringOne.incrby(region, null, 3);
+
+    verify(region).put(any(), any());
+    assertThat(stringOne.hasDelta()).isFalse();
+  }
+
+  @Test
+  public void incrbyfloatStoresStableDelta() {
+    Region<RedisKey, RedisData> region = uncheckedCast(mock(Region.class));
+    final byte[] baseBytes = {'1'};
+
+    when(region.put(any(), any()))
+        .thenAnswer(invocation -> validateDeltaSerialization(baseBytes, invocation));
+    RedisString stringOne = new RedisString(baseBytes);
+
+    stringOne.incrbyfloat(region, null, new BigDecimal(3.0));
```

Comment:
IntelliJ has a warning here about a double value being passed into the BigDecimal constructor. While it's not really relevant in this test, it can be fixed by passing in a String instead. The same warning is present on line 246.